src/prommis/uky/tests/test_uky_flowsheet.py

```diff
@@ -44,655 +49,689 @@
+                * model.fs.sl_sep1.retained_liquid_outlet.flow_vol[0],
+                to_units=units.kg / units.hr,
+            )
         )
-        assert value(model.fs.costing.total_BEC) == pytest.approx(5.244, rel=1e-4)
-        assert value(model.fs.costing.total_installation_cost) == pytest.approx(
-            10.33, rel=1e-4
+
+        load_purge = value(
+            units.convert(
+                model.fs.load_sep.purge.conc_mass_comp[0, REE]
+                * model.fs.load_sep.purge.flow_vol[0],
+                to_units=units.kg / units.hr,
+            )
         )
-        assert value(model.fs.costing.other_plant_costs) == pytest.approx(
-            0.0016309, rel=1e-4
+
+        scrub_purge = value(
+            units.convert(
+                model.fs.scrub_sep.purge.conc_mass_comp[0, REE]
+                * model.fs.scrub_sep.purge.flow_vol[0],
+                to_units=units.kg / units.hr,
+            )
         )
-        assert value(model.fs.costing.total_fixed_OM_cost) == pytest.approx(
-            7.2484, rel=1e-4
+
+        precip_purge = value(
+            units.convert(
+                model.fs.precip_sep.purge.conc_mass_comp[0, REE]
+                * model.fs.precip_sep.purge.flow_vol[0],
+                to_units=units.kg / units.hr,
+            )
         )
-        assert value(model.fs.costing.total_variable_OM_cost[0]) == pytest.approx(
-            10.7422, rel=1e-4
+
+        roaster_retained_liquid = value(
+            units.convert(
+                model.fs.sl_sep2.retained_liquid_outlet.conc_mass_comp[0, REE]
+                * model.fs.sl_sep2.retained_liquid_outlet.flow_vol[0],
+                to_units=units.kg / units.hr,
+            )
         )
-        assert value(model.fs.costing.land_cost) == pytest.approx(
-            6.1234e-5, rel=1e-4
-        )  # Expression, not Var
-        assert value(model.fs.costing.total_sales_revenue) == pytest.approx(
-            0.00018136, rel=1e-4
+
+        roaster_product = value(
+            units.convert(
+                model.fs.roaster.flow_mol_comp_product[0, REE]
+                * molar_mass[REO]
+                * REE_mass_frac[REO],
+                to_units=units.kg / units.hr,
+            )
         )
-        assert value(
-            model.fs.L_pe_tanks.costing.bare_erected_cost["4.2"]
-        ) == pytest.approx(0.0003690, rel=1e-4)
-        assert value(
-            model.fs.L_tank_mixers.costing.bare_erected_cost["4.3"]
-        ) == pytest.approx(0.08625, rel=1e-4)
-        assert value(model.fs.L_pump.costing.bare_erected_cost["4.4"]) == pytest.approx(
-            0.01446, rel=1e-4
+
+        assert feed == pytest.approx(
+            filter_cake
+            + filter_cake_liquid
+            + load_purge
+            + scrub_purge
+            + precip_purge
+            + roaster_retained_liquid
+            + roaster_product,
+            rel=1e-4,
```

Comment:
This is a looser tolerance than I would normally like to see - does it pass with a tolerance of 1e-5?

Reply:
It does not pass with a tolerance of 1e-5

Reply:
What is the difference, and do we have any idea where in the process the differences is arising?

Reply:
I can look into where the difference is arising if we want to get this to pass with 1e-5 tolerance.
![image](https://github.com/prommis/prommis/assets/96305519/6ad0ff57-ad24-4018-ad71-37347ed0e7ec)

Reply:
Umm... you need to look into that. The relative error in those values is 1%, which is huge.

Reply:
@andrewlee94 I've addressed this and changed the tolerance to 1e-7. I wasn't accounting for the dust stream in the roaster.